forum/profiles/models.py

```diff
@@ -1,3 +1,72 @@
 from django.db import models
+from django.core.validators import MinValueValidator
+from phonenumber_field.modelfields import PhoneNumberField
+from django.contrib.auth import get_user_model
 
 # Create your models here.
+
+User = get_user_model()
+
+class StartupProfile(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="startup_profile", db_index=True)
+    company_name = models.CharField(max_length=200)
+    industry = models.CharField(max_length=100)
+    size = models.CharField(max_length=50)
+    country = models.CharField(max_length=50)
+    city = models.CharField(max_length=50)
+    zip_code = models.CharField(max_length=20)
+    address = models.CharField(max_length=250, blank=True, null=True)
+    phone = PhoneNumberField(null=True, blank=True)
+    email = models.EmailField(unique=True, db_index=True)
```

Comment:
If you’re using custom validation logic or want better clarity, consider adding a validator to check email format explicitly.
```
from django.core.validators import validate_email

email = models.EmailField(
    unique=True,
    db_index=True,
    validators=[validate_email]
)
```